packages/next/src/server/app-render/entry-base.ts

```diff
@@ -10,8 +10,8 @@ export {
 // eslint-disable-next-line import/no-extraneous-dependencies
 export { prerender } from 'react-server-dom-webpack/static'
 
-// TODO: Just re-export `* as ReactServer`
-export { captureOwnerStack, createElement, Fragment } from 'react'
+import React from 'react'
```

Comment:
```suggestion
import * React from 'react'
```

There's no default export. Default import only works because of esmInterop handlers being injected.